apartment/api/serializers.py

```diff
@@ -125,3 +128,19 @@ class ProjectDocumentSerializer(serializers.Serializer):
     estate_agent_phone = serializers.CharField(source="project_estate_agent_phone")
     coordinate_lat = serializers.FloatField(source="project_coordinate_lat")
     coordinate_lon = serializers.FloatField(source="project_coordinate_lon")
+    installment_templates = serializers.SerializerMethodField()
+
+    def __init__(self, *args, **kwargs):
+        # the actual "many" is not available here so we need this stupid workaround
```

Comment:
Clean comments, please. Things that seem funny at the moment are just confusing later.

Reply:
Hmm, I wasn't actually trying to be funny, but if you are referring to the word "stupid" there, I can partly agree that it was a bit stupid word choice. 😄 Anyways, the comment isn't needed anymore.